src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -1,13 +1,49 @@
-<ng-container *ngIf="{upgradeInfo: upgradeInfo$ | async, error: upgradeInfoError$ } as upgrade">
-  <div class="row h-25 ms-1"
-       *ngIf="!upgrade.upgradeInfoError && upgrade.upgradeInfo as upgradeInfo; else checkUpgrade">
-    <ng-container *ngIf="healthData$ | async as healthData">
-      <div class="col-lg-3 h-50 d-flex flex-column border justify-content-center align-items-center">
-        <span class="bold">Current Version</span>
-        <span class="mt-1">{{ version }}</span>
+<div class="row h-25"
+     *cdScope="'configOpt'">
+  <ng-container *ngIf="healthData$ | async as healthData">
+    <cd-card class="col-sm-3 px-3 d-flex"
+             cardTitle="Upgrade Status"
```

Comment:
It's a tiny change but it's consistent (next card is called "Current version") and it's extremely actionable:
```suggestion
             cardTitle="New Version"
```
```suggestion
             cardTitle="New Version Available"
```
As a reference:
![image](https://github.com/ceph/ceph/assets/37327689/f05661fe-4748-497b-a74a-8ea77555c079)

Reply:
for Now I went with `New Version` because putting `Available` means that I'll have to put some extra conditions to handle the title when there's no update Available!